Sources/Rendering/OpenGL/ShaderProgram/index.js

```diff
@@ -184,6 +184,26 @@ export function vtkShaderProgram(publicAPI, model) {
     return true;
   };
 
+  publicAPI.setUniform4f = (name, v) => {
+    const location = publicAPI.findUniform(name);
+    if (location === -1) {
+      model.error = `Could not set uniform ${name} . No such uniform.`;
+      return false;
+    }
+    model.context.uniform4f(location, v[0], v[1], v[2], v[3]);
+    return true;
+  };
+
+  publicAPI.setUniform4fv = (name, count, v) => {
+    const location = publicAPI.findUniform(name);
+    if (location === -1) {
+      model.error = `Could not set uniform ${name} . No such uniform.`;
+      return false;
+    }
+    model.context.uniform4fv(location, count, v);
```

Comment:
I just copied the code of `setUniforms3fv()` but it looks like you are right, there is no overload with such a `count` parameter for any of them.

But then I am not sure that your fix is correct because the `count` parameter is not used but I am not sure what it is supposed to do as the webgl function is just `void gl.uniform[1|2|3|4]fv(location, value);`